Formula/wine.rb

```diff
@@ -309,11 +309,6 @@ def install
       end
 
       resource("freetype").stage do
-        # Enable sub-pixel rendering
-        inreplace "include/freetype/config/ftoption.h",
-                  "/* #define FT_CONFIG_OPTION_SUBPIXEL_RENDERING */",
-                  "#define FT_CONFIG_OPTION_SUBPIXEL_RENDERING"
-
         system "./configure", "--prefix=#{libexec}",
                               "--disable-static",
```

Comment:
Does it need the `--enable-freetype-config` option that the main Freetype formula uses? After upgrading, fonts are looking pixellated for me.

Reply:
Can you try with the option, and see if it changes things?

Reply:
No, it didn't change anything; still no subpixel rendering with that.